feature-libs/user/profile/components/my-account-v2/email/my-account-v2-email.component.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { UntypedFormGroup } from '@angular/forms';
+import { Observable } from 'rxjs';
+import { MyAccountV2EmailComponentService } from './my-account-v2-email-component.service';
+import { GlobalMessageType, User } from '@spartacus/core';
+
+import { UserProfileFacade } from '@spartacus/user/profile/root';
+import { filter } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-my-new-account-v2-email',
+  templateUrl: './my-account-v2-email.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  host: {},
+})
+export class MyAccountV2EmailComponent implements OnInit {
+  form: UntypedFormGroup = this.emailComponentservice.form;
+  isUpdating$: Observable<boolean> = this.emailComponentservice.isUpdating$;
+  isEditing: boolean;
+  showingAlert: boolean;
+
+  user$ = this.userProfile
+    .get()
+    .pipe(filter((user): user is User => Boolean(user)));
+  globalMessageType = GlobalMessageType;
+
+  constructor(
+    protected emailComponentservice: MyAccountV2EmailComponentService,
```

Comment:
[MEDIUM]
typo. let's rename `emailComponentservice` to `emailComponenService`

Reply:
fixed